CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.2.6] - 2021-09-28
```

Comment:
you missed a link at the bottom for the diff

Reply:
🤦‍♀️